tests/gpu_benchmark/modal_gpu_test.py

```
@@ -0,0 +1,171 @@
+    env = os.environ.copy()
+    env["PATH"] = "/root/.cargo/bin:" + env.get("PATH", "")
+    
+    results = {}
+    
+    # Test 1: CPU-only baseline
+    print("=" * 60)
+    print("TEST 1: CPU Baseline (8 cores)")
+    print("=" * 60)
+    
+    result = subprocess.run(
+        ["bash", "-c", "source $HOME/.cargo/env && cargo run --release --features cpu 2>&1"],
```

Ruff (0.14.14)

Starting a process with a partial executable path

(S607)